generate_figures.py

```diff
@@ -147,6 +147,9 @@ def xFuncFormatter(x, pos):
     handles, labels = plt.gca().get_legend_handles_labels()
 
     order = argsort([natural_order(label) for label in labels])
+    if reversed_legend:
+        order = list(reversed(order))
```

Comment:
We could just use `order.reverse()` and make it in-place, don't bother though if it's too much effort

Reply:
if I remember correctly I tried this but it didn't work.
anyways, I will merge it ;)